experimental/BasisLieHighestWeight/src/UserFunctions.jl

```diff
@@ -742,3 +742,115 @@ function basis_lie_demazure_ffl(
   # simple roots at the right end speed up the program very much
   return basis_lie_highest_weight_compute(V, operators, monomial_ordering)
 end
+
+@doc raw"""
+    basis_coordinate_ring_kodaira_demazure(type::Symbol, rank::Int, highest_weight::Vector{Int}, weyl_group_elem::Vector{Int}, degree::Int; monomial_ordering::Symbol=:degrevlex)
+    basis_coordinate_ring_kodaira_demazure(type::Symbol, rank::Int, highest_weight::Vector{Int}, weyl_group_elem::Vector{Int}, degree::Int, birational_sequence::Vector{Int}; monomial_ordering::Symbol=:degrevlex)
+    basis_coordinate_ring_kodaira_demazure(type::Symbol, rank::Int, highest_weight::Vector{Int}, weyl_group_elem::Vector{Int}, degree::Int, birational_sequence::Vector{Vector{Int}}; monomial_ordering::Symbol=:degrevlex)
+
+Compute monomial bases for the degree-truncated coordinate ring (for all degrees up to `degree`) 
+of the Kodaira embedding of the generalized flag variety into the projective space of the Demazure module
+with extremal weight `highest_weight * weyl_group_elem` for a simple Lie algebra $L$ of type `type` and rank `rank`.
```

Comment:
I am not sure if this is a mathematically sane description (for the kodaira demazure case). @gfourier could you explain what this actually computes (in a mathematical sense) and suggest some wording for this docstring?

Reply:
```suggestion
Compute monomial bases for the degree-truncated coordinate ring (for all degrees up to `degree`) 
of the Kodaira embedding of a Schubert variety into the projective space of the Demazure module
of extremal weight `highest_weight * weyl_group_elem` associated with a simple Lie algebra $L$ of type `type` and rank `rank`.
```
this is what Ghislain wrote me on zoom (with slightly adapted language and consistent to our notation.
could you please adapt this also for the ffl one?